docs/source-2.0/guides/generating-code/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _generating-code:
```

Comment:
Should we link this page at the end of the quickstart guide?

Reply:
I added it as a 'next step'. Let me know what you think about that.